index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const teams = require('./teams')
+
+const app = express()
+
+app.get('/teams', (request, response) => {
+  return response.send(teams)
+})
+
+app.get('/:teamId', (request, response) => {
+  const { teamId } = request.params
+
+  const findTeamId = teams.filter((team) => team.id === parseInt(teamId))
```

Comment:
Find might be a better choice here since an ID will only ever lead to one team. Filter returns an array.